apps/frontend/src/components/global/upload_tabs/EditEvaluationModal.vue

```diff
@@ -0,0 +1,187 @@
+              </v-card-title>
+
+              <v-card-text>
+                <v-container>
+                  <v-col>
+                    <v-text-field
+                      v-model="activeTag.value"
+                      label="Tag Name"
+                      @keyup.enter="commitTag()"
+                    />
+                  </v-col>
+                </v-container>
+              </v-card-text>
+
+              <v-card-actions>
+                <v-spacer />
+                <v-btn color="blue darken-1" text @click="newTagDialog = false">
+                  Cancel
+                </v-btn>
+                <v-btn color="blue darken-1" text @click="commitTag()">
+                  Save
+                </v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+          <v-data-table
+            :headers="headers"
+            :items="activeEvaluation.evaluationTags"
+            disable-sort
+          >
+            <template #[`item.value`]="{item}">
+              <v-edit-dialog
+                :return-value.sync="item.value"
+                large
+                @save="updateTag(item)"
+              >
+                <div>{{ item.value }}</div>
+                <template #input>
+                  <div class="mt-4 title">Update {{ item.value }}</div>
+                  <v-text-field
+                    v-model="item.value"
+                    label="Edit"
+                    single-line
+                    autofocus
+                  />
+                </template>
+              </v-edit-dialog>
+            </template>
+            <template #[`item.actions`]="{item}">
+              <v-icon small @click="deleteTag(item)">mdi-delete</v-icon>
+            </template>
+          </v-data-table>
+        </v-container>
+      </v-card-text>
+
+      <v-card-actions>
+        <v-spacer />
+        <v-btn
+          color="blue darken-1"
+          text
+          blue
+          @click.native="$emit('closeEditModal')"
+        >
+          Cancel
+        </v-btn>
+        <v-btn color="darken-1" text @click="updateEvaluation()">Save</v-btn>
+      </v-card-actions>
+    </v-card>
+  </Modal>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import Modal from '@/components/global/Modal.vue'
+import {Prop} from 'vue-property-decorator';
+import {SnackbarModule} from '@/store/snackbar';
+import {EvaluationModule} from '@/store/evaluations';
+import {IEvaluation} from '@heimdall/interfaces';
+import axios from 'axios';
+
+@Component({
+  components: {
+    Modal
+  }
+})
+export default class EditEvaluationModal extends Vue {
+  @Prop({default: false}) visible!: boolean;
+  @Prop({type: Object, required: true}) readonly active!: IEvaluation;
+
+  newTagDialog: boolean = false;
+  deleteTagDialog: boolean = false;
+  awaitingFinishTyping: boolean = false;
+  activeTag: any = {
+    id: '-1',
+    value: ''
+  };
```

Comment:
Could this be converted to a full blown `EvaluationTag` type? (some fields will need to be added here but that's probably okay, I'd rather the extra fields over the `any` type)